<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDynAlarmBarGenDlg</class>
 <widget class="QWidget" name="QDynAlarmBarGenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>其他</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>移动速度：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Speed">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>33</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x100ms</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>方向</string>
    </property>
    <widget class="QRadioButton" name="Right">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>向右</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Left">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>向左</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>报警排序</string>
    </property>
    <widget class="QRadioButton" name="Time">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>按时间顺序</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Time_Re">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>按时间逆序</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>文本</string>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字体大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文 本 色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Text_Color">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="FontSize">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>label_16</zorder>
   <zorder>label_6</zorder>
   <zorder>Text_Color</zorder>
   <zorder>FontSize</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QLineEdit" name="ID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>选择报警组</string>
   </property>
   <widget class="QPushButton" name="btn_groupset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>报警组设定</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>外观</string>
   </property>
   <widget class="QComboBox" name="Pattern">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>151</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Frame_Color">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>28</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FG_Color">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>69</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>边框色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>151</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>样  式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>111</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>背景色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BG_Color">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>71</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>前景色：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ID</tabstop>
  <tabstop>Pattern</tabstop>
  <tabstop>FontSize</tabstop>
  <tabstop>Speed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
